src/vary.cls.php

```diff
@@ -731,6 +731,19 @@ private function _get_cookie_val($key)
 		return false;
 	}
 
+	private function test_cookie_path($path)
+	{
+		if ($path[0] != '/') {
```

Comment:
This may function better.
```suggestion
		if (substr($path, 0, 1) !== '/') {
```

Reply:
fixed. thank you! :)